### PR TITLE
Test Coverage Increase

### PR DESCRIPTION
**Pull request by nikola-rados**
Tests have been added to all testing scripts bringing ```crmprtd``` coverage to 89%.

**Things to note:**
- ```crmprtd/wmb_exceptions.py``` has been omitted from the coverage check
- the branch was rebased onto the updated ```master``` branch to collect pep8 changes
- further pep8 formatting was done to pass travis testing
- ```testfixtures``` added to ```test_requirements.txt```

### CODE REVIEW

File: tests/test_wmb.py [hunk 1]
```diff
@@ -87,10 +119,104 @@ def test_check_history(test_session):
...
+
 
 def test_process(test_session, test_data):
     o = ObsProcessor(test_session, test_data, 1000)
     o.process()
 
     q = test_session.query(Obs)
     assert q.count() == 8
+
+    # test UniquenessError handle
```
Comment (jameshiebert) [inline tests/test_wmb.py]: New test case.

File: tests/test_wmb.py [hunk 2]
```diff
@@ -87,10 +119,104 @@ def test_check_history(test_session):
...
+    o.process()
+    assert o._obs_in_db == 5
+
+
+def test_parse_time_error_handle(test_session, test_data):
```
Comment (jameshiebert) [inline tests/test_wmb.py]: Good, this test case is descriptive in its use of bad data and tests unique behaviuor.

File: tests/test_wmb.py [hunk 3]
```diff
@@ -87,10 +119,104 @@ def test_check_history(test_session):
...
+    assert len(result_single) == 1
+
+
+def test_archive_station(test_session, test_data):
+    o = ObsProcessor(test_session, test_data, 1000)
+    check = o._archive_station('11')
+    assert check == 5
+
+
+class ArgsForTest:
```
Comment (jameshiebert) [inline tests/test_wmb.py]: I think you want to inherit from `object` here.

**Comment by jameshiebert** [comment]
Hi @nikola-rados Are you done addressing feedback on this branch and ready to have this merged?

**Comment by nikola-rados** [comment]
Hello @jameshiebert yes they are ready to go.

**Comment by jameshiebert** [comment]
Yay! OK, you'll want to merge this branch into `dev` as well, so that you can utilize this testing work moving forward with the refactor.

**Comment by nikola-rados** [comment]
Sounds good, I'll get right on it!

**Pull request merged**